src/terrain/routes/schemas/instantlaunches.clj

```diff
@@ -0,0 +1,28 @@
+(ns terrain.routes.schemas.instantlaunches
+  (:use [common-swagger-api.schema :only [describe NonBlankString]]
+        [schema.core :only [defschema Any Keyword optional-key]])
+  (:import [java.util UUID]))
+
+(defschema InstantLaunch
+  {:id              (describe UUID "The UUID of the instant launch")
+   :quick_launch_id (describe UUID "The UUID for the quick launch used for the instant launch")
+   :added_by        (describe String "The username of the user that created the instant launch")
+   :added_on        (describe String "The date and time when the instant launch was created")})
```

Comment:
As much as I hate to give up `snake_case`, should we bite the bullet and start using `camelCase` for new response bodies now?

Reply:
I can probably change it up later, but that's an issue that will cascade through multiple services.